tools/visualstudio.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Visual Studio
+layout: post
+category: tool
+sortReleasesBy: "cycleShortHand"
+
+# A list of releases, supported or not
+# Newer releases go on top of the list, in order
+releases:
+  - releaseCycle: "2019 version 16.11"
+    eol: 2029-04-01
+    cycleShortHand: 20191611
+  - releaseCycle: "2019 version 16.10"
+    eol: false
+    cycleShortHand: 20191610
+  - releaseCycle: "2019 version 16.9"
+    eol: 2022-10-01
+    cycleShortHand: 20191609
+  - releaseCycle: "2019 version 16.8"
+    eol: true
+    cycleShortHand: 20191608
+  - releaseCycle: "2019 version 16.7"
+    eol: 2020-04-01
+    cycleShortHand: 20191607
+  - releaseCycle: "2019 version 16.6"
+    eol: true
+    cycleShortHand: 20191606
+  - releaseCycle: "2019 version 16.5"
+    eol: true
+    cycleShortHand: 20191605
+  - releaseCycle: "2019 version 16.4"
+    eol: 2021-10-01
+    cycleShortHand: 20191604
+  - releaseCycle: "2019 version 16.3"
+    eol: true
+    cycleShortHand: 20191603
+  - releaseCycle: "2019 version 16.2"
+    eol: true
+    cycleShortHand: 20191602
+  - releaseCycle: "2019 version 16.1"
+    eol: true
+    cycleShortHand: 20191601
+  - releaseCycle: "2019 version 16.0"
+    eol: true
+    cycleShortHand: 20191600
+  - releaseCycle: "2017"
+    eol: 2027-04-01
+    cycleShortHand: 2017
+  - releaseCycle: "2015"
+    eol: 2025-10-01
+    cycleShortHand: 2015
+  - releaseCycle: "2013"
+    eol: 2024-01-01
+    cycleShortHand: 2013
+  - releaseCycle: "2012"
+    eol: 2023-01-01
+    cycleShortHand: 2012
+  - releaseCycle: "2010"
+    eol: true
+    cycleShortHand: 2010
+    
+iconSlug: visualstudio
+permalink: /visualstudio
+link: https://docs.microsoft.com/en-us/visualstudio/releases/2019/servicing
+activeSupportColumn: false
+releaseColumn: false
+releaseDateColumn: false
+eolColumn: End of Support
```

Comment:
Please change this to "Active Support" because when the value is rendered to "Yes" it means that Support is active and NOT ended.

Reply:
Good one!

Reply:
done!